improver/nbhood/use_nbhood.py

```diff
@@ -192,7 +192,7 @@ def process(self, cube, mask_cube):
                     lead_times=self.lead_times,
                     weighted_mode=self.weighted_mode,
                     sum_or_fraction=self.sum_or_fraction, re_mask=self.re_mask
-                    ).process(x_y_slice, mask_cube=cube_slice)
+                    )(x_y_slice, mask_cube=cube_slice)
```

Comment:
Long init list

Reply:
Took the initialisation out of the loop.